.gitignore

```diff
@@ -39,3 +39,8 @@ yarn*
 # test data for local node tests
 test-data/
 tmp.*
+
+# Ignore files created by deployment scripts
```

Comment:
Perhaps a more local .gitignore in the `devops/infrastructure` folder would be better

Reply:
Done. Thanks for the suggestion. 😄